python/tvm/relay/frontend/tflite.py

```diff
@@ -98,6 +98,7 @@ def __init__(self, model, subgraph, exp_tab):
             'SPACE_TO_BATCH_ND': self.convert_space_to_batch_nd,
             'PRELU': self.convert_prelu,
             'TRANSPOSE_CONV': self.convert_transpose_conv,
+            'DETECTION_POSTPROCESS': self._convert_detection_postprocess
```

Comment:
Please change to `self._convert_detection_postprocess` as other convert function's code style. We should keep the same cod style.